QuickDeployment/V2/QuickDeploymentSetup.md

```diff
@@ -0,0 +1,349 @@
+1. Adjust the hardware specifications of the VM.
+   For a performant system, the following are recommended:
+    * 4 vCPUs
+    * 8 GB RAM
+
+<details>
+    <summary>Cick to show animated summary</summary>
+
+![QDE Hyper-V VHD](images/quickdeploy/QDE-hyperv.gif)
+</details>
+
+### Platform: VMware / Other (OVF/OVA Template)
+
+1. Download either the OVF or OVA from the provided link, and unzip to the directory you'd like to store it.
+1. Import the template into your hypervisor of choice.
+   You can typically directly import the OVA or OVF template file, and most settings should be pre-configured for you.
+    * Instructions on doing this for VMware are [here](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.html.hostclient.doc/GUID-FBEED81C-F9D9-4193-BDCC-CC4A60C20A4E_copy.html).
+1. Once the import completes, go into the VM's hardware settings and expand the main disk to at least 500GB.
+1. Boot up the VM, and (if needed) install any software required / recommended for your hypervisor.
+
+### Platform: VMware (VMDK file)
+
+1. Download VMDK from provided link, and unzip it to the directory you wish to store it.
+2. For ESX/ESXi, open vSphere and upload the downloaded VMDK to your datastore.
+3. Create a new VM.
+4. When prompted for OS type, choose `Windows Server 2019` (if available), or `Windows Server 2016 or later`.
+5. If prompted for boot firmware, choose `Legacy BIOS` (**not** UEFI).
+6. When asked to create a new disk or attach, delete the default disk, select attach, and browse to the VMDK you uploaded. **IMPORTANT**: [vCenter/ESX/ESXi] You **must** select an `IDE controller` under the “Controller Location" setting of the disk. If you leave the controller as SCSI (default), your VM will not boot.
+7. Adjust the hardware specifications of the VM. For a performant system, the following are recommended:
+    - 4 vCPUs
+    - 8 GB RAM
+8. Once you click Finish, go back into the `Edit settings` context menu for the VM, and expand the disk you attached to 500GB (double-check in OS, and extend if needed). **NOTE**: likely you will need to allocate the additional space to the C drive.
+9. Boot up VM, and Install VMware Tools using the console menus (this will require a reboot).
+
+<details>
+    <summary>Click to show animated summary (ESX/i)</summary>
+
+![QDE VMware VMDK](images/quickdeploy/QDE-vmdk-esx.gif)
+</details>
+
+<details>
+    <summary>Click to show animated summary (VMware Fusion, OSX)</summary>
+
+![QDE VMware VMDK](images/quickdeploy/QDE-vmdk-fusion.gif)
+</details>
+
+### Platform: Other
+
+If your hypervisor supports importing from an OVA or OVF, you should be able to import that image in most cases.
+Otherwise, or if you run into issues doing that, we recommend downloading the VMDK file and converting it to a format supported by your hypervisor.
+Please reach out to support if you need assistance.
```

Comment:
Instead of this, I would like to say "Please only look to perform these steps if you know what you are doing. Our support team unfortunately can only support the hypervisors listed here and will not be able to provide assistance in conversion and ensuring the QDE image works in your environment. However once you are set up, our team can support the running image." cc @steviecoaster thoughts?

Reply:
I like this. I don't want to know how to import this stuff into KVM, Nutanix, etc etc.

Reply:
> Please only look to perform these steps if you know what you are doing. We can only support the hypervisors listed here, and will not be able to provide assistance in conversion and ensuring the QDE image works in your environment. However once you are set up, our team can support the running image.

Personally I have a pet peeve about "unfortunately", I just see a lot of discussions go downhill immediately after that word is used 👍

Reply:
I would agree about not using the word 'unfortunately'.